### ISSUE TITLE
Track views with Plausible on the public page

### ISSUE THREAD

**Issue by emilienbidet** [calcom/cal.com#12240]
### Is your proposal related to a problem?

I've been frustrated when I installed the Plausible app this morning.
It tracks event type views but not public page views.

### Describe the solution you'd like

- Moving the options bellow to **Apps > Installed Apps > Plausible**, it's more logical.

![Options](https://github.com/calcom/cal.com/assets/44983373/e1edc207-7dcc-4071-b82a-dc542f13021b)

- Let only the toggle to whetter activate Plausible or not on the **Event Type > App > Plausible**.

- Track public page views with Plausible

### Describe alternatives you've considered

No alternative or workaround found yet.

### Additional context

I can contribute to dev this feature based on the existing code if someone tell me quickly where it should be implemented.

### Requirement/Document

No document required.

---
##### House rules
- If this issue has a `🚨 needs approval` label, don't start coding yet. Wait until a core member approves feature request by removing this label, then you can start coding.
  - For clarity: Non-core member issues automatically get the `🚨 needs approval` label.
  - Your feature ideas are invaluable to us! However, they undergo review to ensure alignment with the product's direction.

**Comment by PeerRich** [comment]
@hariombalhara do you know more here?

**Comment by hariombalhara** [comment]
Yeah, all tracking apps track event type only and not profile page. It has comeup earlier as well.

**Comment by hariombalhara** [comment]
Related issues #8509, #6431, #6564,

**Comment by Shyam-Raghuwanshi** [comment]
hi @emilienbidet  why i am not able to see the `plausible url `
